core/modules/dashboard/config/layout.layout.dashboard.json

```diff
@@ -0,0 +1,353 @@
+{
+    "_config_name": "layout.layout.dashboard",
+    "path": "admin/dashboard",
+    "name": "dashboard",
+    "title": "Dashboard",
+    "description": "Site overview dashboard",
+    "renderer_name": "standard",
+    "module": "dashboard",
+    "weight": 0,
+    "storage": 2,
+    "layout_template": "sutro",
+    "disabled": false,
+    "settings": {
+        "title": "",
+        "title_display": "default",
+        "title_block": null
+    },
+    "positions": {
+        "header": [
+            "1d07f2f1-c523-4c0d-a8ca-e7089aed712b",
+            "24a512d7-bcd7-4d4f-99eb-5f7c750a461d"
+        ],
+        "top": [
+            "eb936b48-d2ec-4d9f-9dd9-ef24a1ad3493"
+        ],
+        "content": [
+            "a144d9d3-0cac-4c20-989e-75220ab686cb"
+        ],
+        "half1": [
+            "5a4ec847-16c1-45dd-b168-58afed5081d4",
+            "8ebebe78-4aa4-4d08-a79f-2463daf79863",
+            "f149b0ac-ff4c-46f5-8402-ee1aac572dfb"
+        ],
+        "half2": [
+            "a4b4935d-e085-4d77-a522-ea805e172c1a",
+            "2496a766-3efd-410d-88ea-c84f7b67806e",
+            "fe51ec3f-6310-42cd-87f1-1c2361d666c1"
+        ],
+        "bottom": [],
+        "footer": [
+            "55aa5247-2948-450b-b65c-0fcff326e1a1"
+        ],
+        "title": []
+    },
+    "content": {
+        "1d07f2f1-c523-4c0d-a8ca-e7089aed712b": {
+            "plugin": "system:header",
```

Comment:
I think the system header block should be removed, to match the default administrative layout.